lib/livebook_web/live/hub/edit_live.ex

```diff
@@ -88,12 +82,12 @@ defmodule LivebookWeb.Hub.EditLive do
   end
 
   @impl true
-  def handle_info({:hub_connected, id}, %{assigns: %{hub: %{id: id}}} = socket) do
-    {:noreply, push_navigate(socket, to: ~p"/hub/#{id}")}
+  def handle_info({:hub_connected, id}, %{assigns: %{hub: %{id: id}, params: params}} = socket) do
+    {:noreply, push_patch(socket, to: ~p"/hub/#{id}?#{params}")}
```

Comment:
Actually, why are we doing any kind of navigation in this case, is it something related to flash?

Reply:
Good call. I think we just need to call handle_params again. I will refactor.